databases/week1/homewrk.sql

```diff
@@ -0,0 +1,82 @@
+Use hyf_lesson1;
+/* Find out how many tasks are in the task table */
+select
+  COUNT(id)
+from
+  task;
+  /* Find out how many tasks in the task table do not have a valid due date */
+select
+  COUNT(id)
+from
+  task
+where
+  IsNull(due_date);
+  /* Find all the tasks that are marked as done */
+select
+  *
+from
+  task
+where
+  status_id = 3;
+  /* Find all the tasks that are not marked as done */
+select
+  *
+from
+  task
+where
+  status_id != 3;
+  /* Get all the tasks, sorted with the most recently created first */
+select
+  *
+from
+  task
+ORDER BY
+  created DESC;
+  /* Get the single most recently created task */
+select
+  *
+from
+  task
+ORDER BY
+  created DESC
+LIMIT
+  1;
+  /* Get the title and due date of all tasks where the title or description contains database */
+select
+  *
```

Comment:
Here, keep in mind that not all the information is required. 
Select the specific fields mentioned in the task. 👍